src/components/EtherscanLink/index.jsx

```diff
@@ -1,29 +1,37 @@
 // @flow
 import React from 'react'
 import { withStyles } from '@material-ui/core/styles'
+import cn from 'classnames'
 import Block from '~/components/layout/Block'
-import Span from '~/components/layout/Span'
+import Paragraph from '~/components/layout/Paragraph'
 import CopyBtn from '~/components/CopyBtn'
 import EtherscanBtn from '~/components/EtherscanBtn'
 import { shortVersionOf } from '~/logic/wallets/ethAddresses'
 import { styles } from './style.js'
+import EllipsisTransactionDetails from '~/routes/safe/components/AddressBook/EllipsisTransactionDetails'
 
 type EtherscanLinkProps = {
   type: 'tx' | 'address',
   value: string,
   cut?: number,
+  knownAddress?: boolean,
   classes: Object,
 }
 
 const EtherscanLink = ({
-  type, value, cut, classes,
+  type, value, cut, classes, knownAddress,
 }: EtherscanLinkProps) => (
   <Block className={classes.etherscanLink}>
-    <Span size="md">
+    <Paragraph
+      size="md"
+      noMargin
+      className={cn(knownAddress && classes.addressParagraph)}
+    >
       {cut ? shortVersionOf(value, cut) : value}
-    </Span>
+    </Paragraph>
```

Comment:
Why was this changed from `<span>` to `<p>`?

Reply:
Fixed

Reply:
the reason I changed it from p to span while ago is that we'd nest this component into other paragraphs, and paragraph cannot be nested to paragraph